apollo-router/src/query_planner/dual_query_planner.rs

```diff
@@ -87,8 +87,11 @@ impl BothModeComparisonJob {
 
             let start = Instant::now();
 
+            // TODO pass conditions
             // No question mark operator or macro from here …
-            let result = self.rust_planner.build_query_plan(&self.document, name);
+            let result = self
```

Comment:
@lrlna @goto-bus-stop any idea how where do I get the override conditions from? looks like we are not passing down the plan options in dual qp mode?

Reply:
0e9e6c5